health-services/project-factory/src/server/utils/transforms/searchResponseConstructor.ts

```
@@ -80,6 +80,7 @@ export const generatedResourceTransformer = (dbRows: any[] = []) => {
     item.additionalDetails = item.additionaldetails;
     item.additionalDetails.Filters = item?.additionaldetails?.filters ? {} : item?.additionaldetails?.filters;
     item.fileStoreid = item.filestoreid;
+    item.locale = item.locale;
```

Biome (1.9.4)

[error] locale is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)